src/test/java/com/neo4j/docker/TestPluginInstallation.java

```diff
@@ -0,0 +1,140 @@
+package com.neo4j.docker;
+
+import com.neo4j.docker.utils.ExampleNeo4jPlugin;
+import com.neo4j.docker.utils.HostFileHttpHandler;
+import com.neo4j.docker.utils.HttpServerRule;
+import com.neo4j.docker.utils.JarBuilder;
+import com.neo4j.docker.utils.SetContainerUser;
+import com.neo4j.docker.utils.TestSettings;
+import org.junit.Rule;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.migrationsupport.rules.EnableRuleMigrationSupport;
```

Comment:
You're using a mix of JUnit4 and JUnit5 here. We've been using JUnit5 in the rest of the tests.
JUnit4 Rules are better imho, I don't know how you want to do it here. But you should be consistent at least.

Reply:
If you decide to keep this JUnit4 then we should migrate the rest of the tests back to junit4. But that can be a seperate PR. 
I think there's migration packages you'll need to add to the pom if you keep it junit4:
https://junit.org/junit5/docs/current/user-guide/#migrating-from-junit4-running

Reply:
I've added the migration package so that I can still use the `ExternalResource` `Rule` for the HTTP Server